### DISCUSSION TITLE
RTOS for the MiV

### DISCUSSION THREAD

**Discussion by rockoguido** [polarfire-soc/polarfire-soc-documentation#199] [Bare metal embedded software] [answered]
Hi. I hope this is the right forum, though I know this is SoC focused.    Got a user targeting the RPG4 looking for a RTOS
 running on the MiV.   Told them FreeRTOS and perhaps Zephyr but wanted to check.. I believe FreeRTOS is the only legit option.. 
Thx
Dave

**Comment by nitindeshpande** [comment, marked as answer]
@rockoguido 
Please post this question at https://github.com/orgs/Mi-V-Soft-RISC-V/discussions/categories/embedded-software
